tests/test_client.py

```diff
@@ -53,12 +53,12 @@
 }
 
 
-class TestDevnetClient(TestCase):
+class Testnet(TestCase):
```

Comment:
theres already a sbclass for testnet so this is kind of uncessary.

Reply:
my only reasoning was to make pytest output cleaner:

```
tests/test_client.py::Testnet::test_cancel_and_replace_order
tests/test_client.py::Testnet::test_cancel_order
```

instead

```
tests/test_client.py::TestDevnetClient::test_cancel_and_replace_order
tests/test_client.py::TestDevnetClient::test_cancel_order
```

can revert if you prefer